### PR TITLE
added darglint to `make lint` command

### PR DESCRIPTION
**Pull request by solarw**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -248,6 +248,7 @@ lint:
 	isort $(PYCOSM_SRC_DIR) $(PYCOSM_TESTS_DIR) $(PYCOSM_EXAMPLES_DIR) setup.py
 	flake8 $(PYCOSM_SRC_DIR) $(PYCOSM_TESTS_DIR) $(PYCOSM_EXAMPLES_DIR) setup.py
 	vulture $(PYCOSM_SRC_DIR) $(PYCOSM_TESTS_DIR) $(PYCOSM_EXAMPLES_DIR) setup.py --exclude '*_pb2.py,*_pb2_grpc.py'
+	bash -c "find $(PYCOSM_SRC_DIR) $(PYCOSM_TESTS_DIR) $(PYCOSM_EXAMPLES_DIR) -iname '*.py'|grep -v cosmpy/protos|xargs darglint"
```
Comment (5A11) [inline Makefile:251]: This is a bit messy. Couldn't you add it cleaner like in the aea framework https://github.com/fetchai/agents-aea/blob/main/Makefile?

Comment (5A11) [inline Makefile:251]: This is a bit messy. Couldn't you add it cleaner like in the aea framework https://github.com/fetchai/agents-aea/blob/main/Makefile?

File: Pipfile [hunk 2]
```diff
@@ -5,6 +5,17 @@ verify_ssl = true
...
+bandit = "==1.7.0"
+black = "==21.6b0"
+isort = "==5.9.3"
+flake8 = "==3.9.2"
+mypy = "==0.910"
+pyflakes = "==2.3.1"
+pytest = "==7.0.1"
+tox = "==3.24.1"
+vulture = "==2.3"
+
```
Comment (5A11) [inline Pipfile:18]: At some point we decided to add all dev dependencies in setup.py and exclude it from Pipfile. You can see dangling is listed in setup.py already.

Comment (solarw) [reply Pipfile]: fixed

**Review by 5A11 — APPROVED** [review]
lgtm

File: Makefile (hunk 1 above)
Comment (solarw) [reply Makefile]: done

**Pull request merged**